Sources/StreamChat/Database/DTOs/PollVoteDTO.swift

```diff
@@ -75,7 +84,7 @@ extension PollVoteDTO {
 extension NSManagedObjectContext {
     @discardableResult
     func savePollVotes(payload: PollVoteListResponse, query: PollVoteListQuery?, cache: PreWarmedCache?) -> [PollVoteDTO] {
-        let isFirstPage = query?.pagination.offset == 0
+        let isFirstPage = query?.pagination.cursor == nil
```

Comment:
In other places I am seeing:
`pagination.offset == 0 && pagination.cursor == nil`
Maybe we should keep it like that to be sure?